lib/utils.js

```diff
@@ -111,6 +112,7 @@ exports.fuse = function fuse (target, handlers) {
 // merges a object's proppertys / values with a other object
 exports.merge = function merge (target, obj) {
   for (var i in obj) {
```

Comment:
In fact, the formal way of `for ... in` in JSHint is just wrapping the body in `if(foo.hasOwnProperty(bar))`.

If you don't, JSHint will show a warning. And of cause, this place made some bugs.